### PR TITLE
Mutant bears with mutant offal

### PR DESCRIPTION
**Pull request by Coolthulhu**
#### Summary

SUMMARY: [Content] "Mutant bears with mutant offal"

#### Purpose of change

Flesh out post-apocalypse forest a tiny bit.

#### Describe the solution

Bears now evolve into scaled bears, which have chitin instead of fur and have mutant offal instead of lungs, liver and kidneys.
Mutant offal is usable in some recipes that require non-specific offal: haggis, humble pie, pickled offal.
It can also be fried on its own.
In all cases, it is about double as mutagenic as meat.

Rethemed mutant meat: it no longer has hair or bones, instead it is tough and smells like chemicals. Non-specific chemicals, because it's supposed to differ between individuals, but is supposed to encompass things like burnt sulfur, rotten eggs, bleach, bitter almonds, freshly cut grass, musty hay, salt spray, artificial apple flavoring, acetone and similar smells you wouldn't expect from a dead mammal.
Mutant organs are themed as being too different between individuals to properly categorize.

### CODE REVIEW

**Comment by chaosvolt** [comment]
Hopefully this won't have too much risk of setting a trend like DDA's following, where all non-mutagenic sources tend to eventually evolve into mutants. Though I can see that being a useful way to put pressure on the player as the game progresses, it probably wouldn't be good to have it become a full halt to meat-eating like it tends to be in DDA (since almost every animal in the game has mutant evolution).

**Comment by Coolthulhu** [comment]
> Hopefully this won't have too much risk of setting a trend like DDA's following, where all non-mutagenic sources tend to eventually evolve into mutants.

It is my intent to have renewable, unmutated sources of food become increasingly scarce. Not just mutant animals, but also plants.
At the moment, food-related mutations just don't matter because unmutated food is plentiful and completely renewable.

**Comment by Proxiehunter** [comment]
> > Hopefully this won't have too much risk of setting a trend like DDA's following, where all non-mutagenic sources tend to eventually evolve into mutants.
> 
> It is my intent to have renewable, unmutated sources of food become increasingly scarce. Not just mutant animals, but also plants. At the moment, food-related mutations just don't matter because unmutated food is plentiful and completely renewable.

If the player domesticates unmutated chicken, cows, sheep, whatever, are these planned to _also_ mutate or can the player with careful management and the skill/luck to domesticate some animals early have a breeding stock of unmutated animals to provide meat and eggs free of mutagenic toxins? The same questions applies to the mutant plants you've mentioned.

Applying pressure like this could be interesting if the player is allowed to farm and provide themselves with non-mutagenic food, it even helps encourage the use of the existing mechanics to grow crops and breed animals. Although if possible I think breeding animals could use something of a re-work so it's less reliant on what I understand is "maybe if there's two or more of them they'll sometimes spawn a baby". Maybe a breeding paddock location that can be constructed that makes spawning a baby animal more likely, with high survival skill maybe some primitive artificial insemination or something.

But that's a bit of a digression. My point is that if the player is capable of _working_ to continue to have access to non-mutated food by using the existing farming system what you're proposing would be interesting to me. If _not_ it seems to make the game harder for the player for no appreciable benefit and could maybe be a mod players add for extra challenge instead.  

Honestly one of my biggest problems with original DDA was continual addition of difficulty for no appreciable benefit to player making the game frustrating for new players or even _returning_ players who suddenly found the game much harder than they recalled it being. Your idea of making unmutated food increasingly scarce runs the risk of being just that if not handled carefully.

**Comment by chaosvolt** [comment]
That's the basic thought I was having, yeah. It'd be a good thing if it makes regular meat harder to obtain, so long as it isn't taken to the extreme of it becoming completely inaccessible. More pressure is good, too much just makes hunting become non-viable.

File: data/json/monsters/mutant_animal.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "proportional": { "hp": 2.0 },
+    "relative": { "morale": 40, "melee_skill": 1, "melee_dice_sides": 2, "melee_cut": 4, "dodge": -2 },
+    "armor_bash": 8,
+    "armor_cut": 14,
+    "armor_stab": 30,
+    "armor_acid": 8,
+    "harvest": "mammal_large_chitin",
+    "delete": { "fear_triggers": [ "SOUND" ] },
+    "//": "Doesn't zombify because it shouldn't regain the fur.",
+    "zombify_into": "mon_null"
```
Comment (chaosvolt) [inline data/json/monsters/mutant_animal.json:17]: We don't have the zombify corpse feature in BN?

Comment (Coolthulhu) [reply data/json/monsters/mutant_animal.json:17]: We do, but I didn't add a zombified variant for this one, so for now it doesn't zombify.

Comment (chaosvolt) [reply data/json/monsters/mutant_animal.json:17]: Oh, when was that added? Is it in widespread use like in DDA, or limited to specific monsters?

Comment (Coolthulhu) [reply data/json/monsters/mutant_animal.json:17]: Dinomod uses it a lot, core critters a bit.

**Comment by chaosvolt** [comment]
I JUST NOW noticed, `offal_raw` doesn't have sweetbread in it.

**Pull request merged**